.circleci/config.yml

```diff
@@ -321,9 +333,15 @@ jobs:
            </table>\
            <h4><p>Boot Nodes</p></h4>\
            <table>\
-            <tr><td>Boot Node 1</td><td>${PRNET_IPS[0]}</td></tr>\
-            <tr><td>Boot Node 2</td><td>${PRNET_IPS[1]}</td></tr>\
-            <tr><td>Boot Node 3</td><td>${PRNET_IPS[2]}</td></tr>\
+            <tr><td>Boot Node 1</td><td>${PRNET_IPS_BOOT_NODES[0]}</td></tr>\
+            <tr><td>Boot Node 2</td><td>${PRNET_IPS_BOOT_NODES[1]}</td></tr>\
+            <tr><td>Boot Node 3</td><td>${PRNET_IPS_BOOT_NODES[2]}</td></tr>\
+           </table>
+           <h4><p>RPC Nodes</p></h4>\
+           <table>\
+            <tr><td>Boot Node 1</td><td>${PRNET_IPS_RPC_NODES[0]}</td></tr>\
```

Comment:
Should also change the labels for each node

Reply:
fixed in: 5362ae76